etc/samples/atoms_generic.sample.yaml

```diff
@@ -0,0 +1,26 @@
+# This schema is not used stand-alone, it is a sub-section of "tendrl_definitions_<sds_nvr>.yaml" | "tendrl_definitions_generic.sample.yaml
+
+---
+sds_version: gluster-3.8.3
+object_details:
+  generic_object:
+    atoms:
+        create:
+          enabled: True
+          uuid: statically generated per atom
+          name: "Name"
+          # List of flows this of which this atom can be part of
+          flows: ["TendrlFlow1"]
+          run: "tendrl.node_agent.atoms.cpu.create.Create"
+          type: "get"
+          pre_run : ["tendrl.node_agent.atoms.<atom_name>.<boolean_function>1"...]
```

Comment:
I think its good add some doc around each attributes here , what each of that means and if there any conditions associated with it. I agree that most of those convey the meaning but good to have